packages/contracts-rfq/test/FastBridgeV2.t.sol

```diff
@@ -44,11 +46,29 @@ abstract contract FastBridgeV2Test is Test, IFastBridgeV2Errors {
     IFastBridgeV2.BridgeParamsV2 internal tokenParamsV2;
     IFastBridgeV2.BridgeParamsV2 internal ethParamsV2;
 
+    bytes internal mockRequestV1;
+    bytes internal invalidRequestV2;
+    bytes internal mockRequestV3;
+
+    function createInvalidRequestV2(bytes calldata requestV2) external pure returns (bytes memory) {
+        // Cut the last 1 byte
+        return requestV2[:requestV2.length - 1];
+    }
```

Comment:
This refactor of the test util contracts is not needed, as it will only pollute the code. The contracts under `test` directory are never deployed, so we don't care about the gas consumption.

As for the slice syntax, the start is not a mandatory field: it defaults to 0.